model-optimizer/extensions/front/mxnet/div_sqrt_dim.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.elementwise import Div
+from mo.front.common.replacement import FrontReplacementOp
+from mo.graph.graph import Graph
+from mo.ops.power import AttributedPower
+from mo.ops.shape import Shape
+from mo.utils.shape import get_shape_values_by_range_idxs
+
+
+class DivSqrtDim(FrontReplacementOp):
```

Comment:
Could you write the description of the operation?

Reply:
There is a description in extensions/ops/div_sqrt_dim.py